ledger/sandbox-on-x/reference.conf

```diff
@@ -342,6 +343,56 @@ ledger {
         #}
       }
 
+      # New style config sample:
+      authentication-domains  {
```

Comment:
I would probably put `domains` inside `authentication` section, i.e. the path would `be authentication.domain`